test/lib/registration/registration.test.ts

```diff
@@ -151,7 +153,7 @@ describe("Registration", () => {
 
   describe("serialising the registration for the API", () => {
```

Comment:
The changes in this test are adding the `BeaconUse` type, which highlighted some inconsistencies that are then fixed.  Other changes are responding to `getMockUse()` returning the `BeaconUse` type (previously `any`).

As a side note, I think `Registration#serialise()` is ripe for simplification and not testing at all (because the type system is the safety net for convert `Type A` to `Type B`, and the logic that's in there _might be_ unnecessary).  But that's an idea for another day 😅